app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/AlertDialogs.kt

```diff
@@ -39,6 +39,38 @@ fun ErrorAlertDialog(
     )
 }
 
+@Composable
+fun ConfirmAlertDialog(
+    title: String,
+    message: String,
+    positiveBtnText: String,
+    negativeBtnText: String,
+    onClickPositiveBtn: () -> Unit,
+    onClickNegativeBtn: () -> Unit
```

Comment:
I think there's no need to write shorthand. 😄 
```suggestion
    positiveButtonText: String,
    negativeButtonText: String,
    onClickPositiveButton: () -> Unit,
    onClickNegativeButton: () -> Unit
```